packages/config/config/devices/0x0371/zwa009.json

```diff
@@ -0,0 +1,210 @@
+// Aeotec ZWA009
+// aerQ Temperature and Humidity Sensor
+{
+  "_approved": true,
+  "manufacturer": "Aeotec Ltd.",
+  "manufacturerId": "0x0371",
+  "label": "ZWA009",
+  "description": "aerQ Temperature and Humidity Sensor",
+  "devices": [
+    {
+      "productType": "0x0002",
+      "productId": "0x0009"
+    }
+  ],
+  "firmwareVersion": {
+    "min": "0.0",
+    "max": "255.255"
+  },
+  "supportsZWavePlus": true,
```

Comment:
Does the lifeline group 1 report the air temperature?
![grafik](https://user-images.githubusercontent.com/17641229/103466112-1fb3f600-4d42-11eb-9a85-31d7bcf8178a.png)

Otherwise it could make sense to list the associations here and configure group 6 as a lifeline too.

Reply:
The docs say that group 1 is the lifeline. Can there be two lifelines? Should I have lifeline on both 1 and 6? @AlCalzone

Reply:
How can I check if group 1 reports the air temperature?

Reply:
Normally there should be only one lifeline. Z-Wave+ mandates that this must be group 1. However some devices report additional values in other groups, so `zwave-js` has the ability to auto-assign the controller to multiple groups instead of just 1.

If you are getting temperature reports without changing anything, then group 1 reports the air temperature.